### ISSUE TITLE
Uploading large files or pastes is not responsive

### ISSUE THREAD

**Issue by brhoades** [brhoades/pasty-http-client#1]
Large files or pastes start uploading when you submit but don't start loading until the server responds. This looks really bad from a UX perspective.

**Comment by brhoades** [comment]
The work is already done for encrypting, decrypting, uploading, and downloading to be done within a webworker. All that needs to be done now is to spin up the first webworker framework and let it go.

**Comment by brhoades** [comment]
Webworker support was added v1.3.0. The site remains responsive now.
